src/ansys/motorcad/core/rpc_client_core.py

```diff
@@ -185,22 +192,30 @@
         if self.reuse_parallel_instances is True:
             self._open_new_instance = False
 
-        if _HAS_PIM and pypim.is_configured():
+        if self._url != "":
+            # Don't want to start a Motor-CAD instance
```

Comment:
No. I'm assuming since a full url is specified that the instance exists. I think in the future I might add open_new_motorcad() and connect_to_motorcad() functions which return a MotorCAD object. I think this class init function/args are starting to get a bit confusing.

Reply:
open_new_instance is True by default. I don't want users to have to specify open_new_instance=False to avoid error if using url. I think separate launch functions are needed really. I've added as an issue: https://github.com/ansys/pymotorcad/issues/142

Reply:
Okay, cheers